### PR TITLE
Fix test email faker

### PR DESCRIPTION
**Pull request by orzechdev**
I want to merge this change because... it fixes test email faker to fix failing e2e tests.

### Screenshots
It replaces the following email whitespaces:
image

### Pull Request Checklist

1. [ ] All visible strings are translated with proper context.
1. [ ] All data-formatting is locale-aware (dates, numbers, and so on).
1. [ ] The changes are tested.
1. [ ] The code is documented (docstrings, project documentation).
1. [ ] Changes are mentioned in the changelog.

### Test Environment Config

API_URI=https://master.staging.saleor.rocks/graphql/

### CODE REVIEW

**Comment by patrys** [comment]
Task linked: [CU-7pr5x2 E2E tests fails due to usage of wrong email addresses.](https://app.clickup.com/t/7pr5x2)

File: cypress/integration/products/buy_a_product.js [hunk 1]
```diff
@@ -4,7 +4,7 @@ import { HEADER_SELECTORS } from "../../elements/main-header/header-selectors";
 import { PRODUCTS_SELECTORS } from "../../elements/products/products-selectors";
 import { CHECKOUT_SELECTORS } from "../../elements/products/checkout-selectors";
 
-const randomWord = faker.random.words(2).replace(" ", "-");
```
Comment (patrys) [inline cypress/integration/products/buy_a_product.js:7]: Should we rely on randomly-generated input for our end-to-end tests?

Comment (michalina-graczyk) [reply cypress/integration/products/buy_a_product.js:7]: @patrys please notice that I am using it for generating a fake email address. Do you have a better idea of how to handle it?

**Review by michalina-graczyk — APPROVED** [review]

File: cypress/integration/products/buy_a_product.js (hunk 1 above)
Comment (patrys) [reply cypress/integration/products/buy_a_product.js:7]: I think we should be fine with using predefined addresses as long as those don't exist in the test database. Debugging tests with random input is a nightmare when they only fail one time out of ten.

Comment (michalina-graczyk) [reply cypress/integration/products/buy_a_product.js:7]: In this case, it is used only for a billing address (required field), the problem was that that previously email address had randomly spaces right now this situation is not gonna happened. I don't see a problem with fake emails here, but of course, we can change that to the same email which is used in login tests but in this scenario, I am gonna get emails every time when the test will run :x . (testers@saleor.io)

Comment (patrys) [reply cypress/integration/products/buy_a_product.js:7]: Do we send confirmation to those emails? If so, they need to be valid emails or we're going to get in trouble with AWS.

Comment (michalina-graczyk) [reply cypress/integration/products/buy_a_product.js:7]: Ok, so I'll change that. FYI https://app.clickup.com/t/6uv31e

**Review by AlicjaSzu — APPROVED** [review]

**Review by dominik-zeglen — APPROVED** [review]

**Pull request merged**